features/map_test.go

```diff
@@ -64,9 +65,25 @@ func TestHaveMapType(t *testing.T) {
 				t.Fatalf("Map type %s isn't supported even though kernel is at least %s: %v", mt.String(), minVersion, err)
 			}
 		})
-
 	}
+}
+
+var mapFlagMinVersion = map[mapFlagCacheEntry]string{
+	{ebpf.Array, unix.BPF_F_MMAPABLE}: "5.5",
+}
+
+func TestHaveMapFlags(t *testing.T) {
+	for entry, minVersion := range mapFlagMinVersion {
+		feature := fmt.Sprintf("%s:%d", entry.mt.String(), entry.flags)
```

Comment:
`entry.flags` will print as a decimal number, we should somehow print the flag as its string value in the test output.